public/js/main.js

```diff
@@ -1,134 +1,116 @@
-appLoading.setColor('#FF9900');
-var cdn_provider_base_url = [];
-var cdn_provider;
-var urlSetDecided = false;
-cdn_provider_base_url.cloudflare = 'https://cdnjs.cloudflare.com/ajax/libs/';
-
-function decideCDNProvider() {
-  var target_cdn_provider = location.hash.substr(1, location.hash.length).toLowerCase();
-  if (!cdn_provider_base_url[target_cdn_provider]) target_cdn_provider = 'cloudflare';
-  return target_cdn_provider;
-}
-
-cdn_provider = decideCDNProvider();
-setFileURLs();
-
-function setFileURLs(new_provider) {
-  if (urlSetDecided === false) {
+/* eslint-env browser */
+/* global Clipboard, SRI, ga, algoliasearch, appLoading, scrollProgress, _, jQuery */
+
+(function ($) {
+  // Set the loading bar color
+  appLoading.setColor('#FF9900');
+
+  /****
+   * CDN Provider Scripting
+   ****/
+
+  var cdn_provider = '', cdn_provider_base_url = {};
```

Comment:
Is var definition merging part of the new coding style?

Reply:
It's much cleaner this way :)